README.md

```diff
@@ -242,6 +242,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Aditya Pandey](https://github.com/adityyaa-10)
   - [Aditya Patel](https://github.com/adityatheoctocatdev)
   - [Aditya Priyadarshi](https://github.com/foxy4096)
+  - [Aditya Rawat](https://github.com/adityasp1)
```

Comment:
Please add your name in line 246, after Aditya Raute.
`U` comes first and later on `W`.